### PR TITLE
PLT-7: Refactoring frontend (chunk 11)

### PR DESCRIPTION
**Pull request by enahum**
- channel view
- Added translations for previous commits
- Fix bug on signup_team email body
- Messages for SocketStore and small fixes
- Add missing translations

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
         "SessionLengthWebInDays": 30,
         "SessionLengthMobileInDays": 30,
         "SessionLengthSSOInDays": 30,
-        "SessionCacheInMinutes": 10
+        "SessionCacheInMinutes": 10,
```
Comment (coreyhulen) [inline config/config.json]: @enahum shouldn't be any changes to config.json, correct?

We have a separate tix to fix this problem with the build environment changing config.json and versiong.go.

Comment (enahum) [inline config/config.json]: Ja, sorry I ran make test and forgot to revert config.json. submitting the fix now

**Comment by coreyhulen** [comment]
@enahum also looks like web/react/utils/locales/en.js contents are deleted?  I think it should be file deleted instead?  (maybe a git diff problem).  I think these files get loaded from `static/js/react-intl-2.0.0-beta-2/locale-data`

**Comment by enahum** [comment]
@coreyhulen I deleted the files not the content on them because they aren't needed anymore.

**Comment by coreyhulen** [comment]
+1

**Comment by hmhealey** [comment]
+1

**Pull request merged**